### PR TITLE
fix: setup build generation and rebuild latest dist

### PR DESCRIPTION
**Pull request by derberg**
Initial investigation told me your code is perfect. Then I started manual testing with `console.log` and noticed there are no logs showing up in the logs 🤔 it can only mean that `dist` is not reflecting what is on the master

After the below change and another build of dist I checked and all works like a charm.
I strongly suggest you configure commit hooks as I suggested in another PR. I did mistakes like this with my actions several times, and then realized that in this case, without a pre-commit hook, it will never stop 😆 

Fix: #80

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "lint": "eslint src",
     "test": "jest src",
-    "build": "rm -rf build && ncc build --minify"
+    "build": "rm -rf build && ncc build index.js -o dist --minify"
```
Comment (amannn) [inline package.json:9]: I also dug a bit into this. It seems like `ncc` has some kind of cache which would result in outdated builds. Even passing `no-cache` to the CLI doesn't make a difference.

But I noticed the above script is wrong: The folder for the built assets isn't called "build", but "dist". It seems like once that is fixed, the script works as expected.

```suggestion
    "build": "rm -rf dist && ncc build"
```

**Review by amannn — COMMENTED** [review]
Thanks for looking into this @derberg!

You're right, the built file had an issue. I wrote a bit more about this in the inline comment.

I'm using a [workflow to automatically release new versions](https://github.com/amannn/action-semantic-pull-request/blob/master/.github/workflows/release.yml) of the action, therefore build artifacts shouldn't be updated in PRs. Can you revert the change to `dist/index.js`? I think with the proposed fix, this should finally work and we can rely that the latest code is in the build artifacts folder 🙂.

**Comment by derberg** [comment]
@amannn ready for the merge. Lint check failing as fresh dist not pushed

**Review by amannn — APPROVED** [review]
Thanks a lot @derberg!

**Pull request merged**